src/compiler/crystal/compiler.cr

```diff
@@ -500,6 +500,19 @@ module Crystal
       else
         link_flags = @link_flags || ""
         link_flags += " -rdynamic"
+
+        if program.has_flag?("freebsd") || program.has_flag?("openbsd")
+          # pkgs are installed to usr/local/lib but it's not in LIBRARY_PATH by
+          # default; we declare it to ease linking on these platforms:
+          link_flags += " -L/usr/local/lib"
+        end
```

Comment:
**thought:** I was wondering about the motivation for this. Seems like the default configuration is just sub-optimal. Apparently compilers in ports do add the folder, only the system compiler does not (https://wiki.freebsd.org/WarnerLosh/UsrLocal).
Maybe we should add some reference for justification here? Not sure if that link would do, it was just the first relevant result on Google.

Reply:
If you build from ports, then a port can patch it in, yeah, but when you compile or cross compile your own crystal binary then it breaks and you must think to export LIBRARY_PATH.

It ain't much, but that's nice. We did it before, but that got lost and I couldn't find why.